client/components/clubs/Tag.tsx

```diff
@@ -0,0 +1,146 @@
+import styled from 'styled-components'
+
+export type Tag = {
+  text: string
+  colour: string
```

Comment:
I vote for the american spelling: `color`

Reply:
hm maybe we should leave this decision up to the incoming FE team